teammapper-frontend/src/app/core/services/map-sync/map-sync.service.ts

```diff
@@ -566,16 +613,18 @@ export class MapSyncService implements OnDestroy {
       this.updateAttachedMap();
     });
 
-    this.mmpService.on('undo').subscribe(() => {
+    this.mmpService.on('undo').subscribe((diff?: any) => {
```

Comment:
always try to avoid any where possible

Reply:
Was a leftover from testing, types already existed so I just updated it now.